simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/VersionInfoExports.java

```diff
@@ -31,8 +31,8 @@
         GaugeMetricFamily jvmInfo = new GaugeMetricFamily(
                 "jvm_info",
                 "JVM version info",
-                Arrays.asList("version", "vendor"));
-        jvmInfo.addMetric(Arrays.asList(System.getProperty("java.runtime.version", "unknown"), System.getProperty("java.vm.vendor", "unknown")), 1L);
+                Arrays.asList("version", "vendor", "runtime"));
+        jvmInfo.addMetric(Arrays.asList(System.getProperty("java.runtime.version", "unknown"), System.getProperty("java.vm.vendor", "unknown"), System.getProperty("java.runtime.name", "unknown")), 1L);
```

Comment:
This line is getting a bit long, can you wrap it?

Reply:
@brian-brazil sure! will also update the javadoc, with the new value ;-)